cpp/example_code/secretsmanager/README.md

```diff
@@ -0,0 +1,37 @@
+# Amazon SecretsManager  code examples for the SDK for C++ using SecretsManagerClient 
```

Comment:
# Secrets Manager code examples for the SDK for C++ using SecretsManagerClient 

(Note: Do we need "using SecretsManagerClient" in the title? (Referring to [template here](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template).) Recommend keeping it only if this README file is a secondary README in addition to a primary Secrets Manager for C++ README.)